test/syntax/idl/extended-attributes.widl

```diff
@@ -9,3 +9,15 @@ interface ServiceWorkerGlobalScope : WorkerGlobalScope {
 // Section 3.11
 [IntAttr=0, FloatAttr=3.14, StringAttr="abc"]
 interface IdInterface {};
+
+// Extracted from http://www.w3.org/TR/2016/REC-WebIDL-1-20161215/#Constructor on 2017-5-18 with whitespace differences
```

Comment:
Note: never take anything from the W3C WebIDL fork. It's obsolete. Please alway use https://heycam.github.io/webidl/